opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/service/TelevisionService.java

```diff
@@ -0,0 +1,70 @@
+package nl.novi.opdrachttechiteasy.service;
+
+
+import jakarta.persistence.EntityNotFoundException;
+import nl.novi.opdrachttechiteasy.exceptions.RecordNotFoundException;
+import nl.novi.opdrachttechiteasy.models.Television;
+import nl.novi.opdrachttechiteasy.repositories.TelevisionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TelevisionService {
+
+
+    @Autowired
+    private final TelevisionRepository televisionRepository;
+
+    public TelevisionService(TelevisionRepository televisionRepository) {
+        this.televisionRepository = televisionRepository;
+    }
+
+
+
+    public List<Television> getTelevisions(){
+        return televisionRepository.findAll();
+    }
+
+    public Television getTelevision (Long id){
+        Optional<Television> optionalTelevision = televisionRepository.findById(id);
+        if (optionalTelevision.isEmpty()) {
+            throw new RecordNotFoundException();
+        } else {
+            return optionalTelevision.get();
+        }
+    }
+
+    public Television saveTelevision (Television television) {
+       return  televisionRepository.save(television);
+    }
+
+    public void removeTelevision(Long televisionId) {
```

Comment:
Deze methode gebruik je nu niet in je controller. Waarschijnlijk omdat het niet werkte? Ik denk dat het probleem daarvoor zit in je PathVariable.